src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -321,4 +321,17 @@ private function registerBusMiddleware(ContainerBuilder $container, string $busI
 
         $container->getDefinition($busId)->replaceArgument(0, new IteratorArgument($middlewareReferences));
     }
+
+    private function getDefinitionClass(string $serviceId, ContainerBuilder $container)
```

Comment:
I think is better set $container as first parameter (to keep consistent with other methods)

```suggestion
    private function getDefinitionClass(ContainerBuilder $container, string $serviceId)
```